CCDB/include/CCDB/BasicCCDBManager.h

```diff
@@ -92,6 +92,15 @@ class BasicCCDBManager
     }
   }
 
+  /// set the object upper validity limit
+  void setUpperValidity(long v) { mUpperValidity = v; }
```

Comment:
@njacazio @Barthelemy I am wondering if the Validity is a good name here, this feature limits the object creation time not the validity...

Reply:
Shall we just call it NotValidAfter as it is in the header?

Reply:
I think so.

Reply:
I agree with the change, I included it in last commit

Reply:
Will you also add the CreatedNotBefore as suggested by @Barthelemy ?

Reply:
@chiarazampolli for this I would prefer opening a second PR if you agree

Reply:
Sure, no problem for me, just let's not forget :-)

Reply:
I think that CreatedNotAfter is good. Also fine to make another PR for the counterpart method.